src/main/java/com/google/jenkins/plugins/k8sengine/KubernetesEnginePublisher.java

```diff
@@ -63,10 +66,13 @@
 public class KubernetesEnginePublisher extends Notifier implements SimpleBuildStep, Serializable {
   private static final Logger LOGGER = Logger.getLogger(KubernetesEnginePublisher.class.getName());
 
+  public static final String ZONE_FILL_ERROR = "Error retrieving Zones";
```

Comment:
This should be moved to Message.properties: https://github.com/jenkinsci/google-kubernetes-engine-plugin/blob/develop/src/main/resources/com/google/jenkins/plugins/k8sengine/Messages.properties

Reply:
Done